rosbridge_library/test/capabilities/test_action_capabilities.py

```diff
@@ -25,7 +25,7 @@
 class TestActionCapabilities(unittest.TestCase):
     def setUp(self):
         rclpy.init()
-        self.executor = SingleThreadedExecutor()
+        self.executor = MultiThreadedExecutor()
```

Comment:
What's the reasoning behind this change?

Reply:
The tests that were previously skipped seem to have needed this. I was getting some flaky behavior without them for some reason.

Reply:
I have very bad experiences with the MultiThreadedExecutor, see https://github.com/ros2/rclpy/issues/1223. So we need to make sure we use CycloneDDS to limit the issues. (Or keep using the SingleThreadedExecutor)

Reply:
They work fine on my end with single threaded executor

Reply:
Alright -- I've put it back into SingleThreaded

That rclpy is issue is very concerning! wow